server/public/model/utils.go

```diff
@@ -872,3 +872,23 @@ func SliceToMapKey(s ...string) map[string]any {
 
 	return m
 }
+
+// LimitRunes limits the number of runes in a string to the given maximum.
+// It returns the original string if it is shorter than the maximum.
+func LimitRunes(s string, maxRunes int) string {
```

Comment:
These functions should return a boolean as well indicating, if the value was trimmed.
In most cases, I would say it would be useful to know, if the value was actually trimmed. And if not then 2nd boolean returned value can simply be skipped.